_pages/zh_TW/godmode9-usage.txt

```diff
@@ -4,7 +4,7 @@ title: "如何使用 GodMode9 "
 
 {% include toc title="條目內容" %}
 
-不論寫腳本、獲得更新訊息或更多訊息，您能加入 [GodMode9 的 Discord 伺服器](https://discord.gg/EGu6Qxw) 以獲得更多幫助。
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
```

Comment:
unnecessary language change again

Did you copy-paste these? If you are going to update the guides other languages, please make sure you don't change the language